### ISSUE TITLE
Webcam in Standby screen

### ISSUE THREAD

**Issue by gielie** [robgeerts/dashticz_v2#49]
I'd like to show my webcam on my standby screen, is this possible?

**Comment by robgeerts** [comment]
Did you try adding it to the standby blocks?
Do you have a temporary url for me where I can acces your webcam? (where I can't see any private stuff ;)

**Comment by gielie** [comment]
Yes I tried to add it to the standby blocks but nothing is showing. I only use the webcam inside my own subnet.

**Comment by HoekieN** [comment]
Maybe because of this issue: https://github.com/robgeerts/dashticz_v2/issues/36
Did you try to also add it to one of the columns of your screens.

**Comment by robgeerts** [comment]
Well, if that is the case, you can easily test it by also adding the webcam to one of the mainscreens. I will fix the bug asap!

**Comment by gielie** [comment]
I do use it in de main screen, and it works perfect.

**Comment by robgeerts** [comment]
Ok, but still not in standby? Then its not the same bug ;)
